topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.md

```diff
@@ -174,19 +186,23 @@ For larger submissions, interactive metadata input can be tedious and not practi
 >
 >      All datasets should now be moved to the *paired section* of the dialog, and the middle column there should show that only the sample accession numbers, *i.e.* `SRR10903401` and `SRR10903402`, will be used as the pair names.
 >
+>    > ### {% icon warning %} Paired collection names
+>    > It is very important that the paired collection names contain no suffix (e.g. _1, _R1, etc.) or file extensions (.fastq, .fastq.gz). The submission tool will add these at runtime and leaving them in the paired collection names will cause a mismatch with the filenames in the metadata table.
+>
```

Comment:
```suggestion
>    > It is very important that the paired collection names contain no suffix (e.g. _1, _R1, etc.) or file extensions (.fastq, .fastq.gz). The submission tool will add these at runtime and leaving them in the paired collection names will cause a mismatch with the filenames in the metadata table.
>    {: .warning }
```

need to close the box or it'll be broken

Reply:
oooh good catch!